### ISSUE TITLE
Keep transaction history when restoring wallet on a new phone

### ISSUE THREAD

**Issue by grmkris** [ACINQ/phoenix#141]
I'd like a way to keep the transaction history or atleast offer the ability to export / import... when changing to a new phone

**Comment by dpad85** [comment]
We will work on that feature in the new version of the app (https://github.com/ACINQ/phoenix-kmm).

**Comment by robbiehanson** [comment]
Transaction history backup has been added to the iOS release (via iCloud). Similar functionality is planned for Android.

**Comment by sr-gi** [comment]
Has this been added in the end?

**Comment by dpad85** [comment]
No, not yet.

**Comment by Sjors** [comment]
I was able to export my history on Android (don't remember the version, but it was a week ago). However I can't import it yet on the iOs side.

**Comment by aassoiants** [comment]
Agree. A simple csv import would be awesome.

**Comment by vlaguz** [comment]
Is this a priority?  Is someone working on it?  Would it be possible if you accept a patch?

**Comment by dpad85** [comment]
Yes, we are working on that here: https://github.com/ACINQ/phoenix/pull/561

If you're a developer, you should be able to build the branch and test it.

For now we are focused on supporting Bolt12, but I will get back to this PR as soon as possible.

**Comment by dpad85** [comment]
Done in #685 for Android, where the export/import must be done manually.

On iOS, the payments history backup is done with iCloud.

**Comment by aassoiants** [comment]
Well done!
